Correct information about plugin enabling (#1760)
ansible/ansible-documentation: docs/docsite/rst/plugins/inventory.rst

   [inventory]
   enable_plugins = host_list, script, auto, yaml, ini, toml, my_plugin

If you use a plugin that supports a YAML configuration source, make sure that the name matches the name provided in the ``plugin`` entry of the inventory source file.

> For other plugins you must either save it in one of the directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>` and enable it or add to a collection and enable it using it's fqcn.

Reviewer: ```suggestion
For other plugins you must either save it in one of the directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>` and enable it or add to a collection and then reference it by FQCN.
```
